src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -384,6 +385,7 @@ uct_dc_mlx5_iface_dci_release(uct_dc_mlx5_iface_t *iface, uint8_t dci_index)
 
     pool->stack_top--;
     pool->stack[pool->stack_top] = dci_index;
+    iface->tx.dcis[dci_index].ep = NULL;
```

Comment:
is this still needed?

Reply:
yes due to https://github.com/openucx/ucx/blob/ab202c97e5d61f166acb9538ab746a36b23bcbc5/src/uct/ib/dc/dc_mlx5_ep.h#L460

Reply:
but isn't setting it to NULL in uct_dc_mlx5_iface_dci_detach() enough?

Reply:
no: dci_release is called from poll_tx->dci_put

Reply:
maybe restore 'iface->tx.dcis[dci_index].ep = NULL;' in dci_put?

Reply:
done